packages/cactus-cmd-api-server/src/test/typescript/integration/jwt-endpoint-authz-scope-enforcement.test.ts

```diff
@@ -72,18 +76,14 @@ test(testCase, async (t: Test) => {
     apiSrvOpts.plugins = [];
     const config = await configService.newExampleConfigConvict(apiSrvOpts);
 
-    const apiServer = new ApiServer({
+    apiServer = new ApiServer({
       config: config.getProperties(),
       pluginRegistry,
     });
-    test.onFinish(async () => await apiServer.shutdown());
-
     const startResponse = apiServer.start();
-    await t.doesNotReject(
-      startResponse,
-      "failed to start API server with dynamic plugin imports configured for it...",
-    );
-    t.ok(startResponse, "startResponse truthy OK");
+
+    expect(startResponse).not.toReject();
```

Comment:
@Leeyoungone This needs to be awaited for.

Reply:
got it! thank you